packages/deployment/app/config.yml

```diff
@@ -146,6 +146,12 @@ default:
       state: ENABLED
       targets:
         - lambda: jobs
+    cronJobs:
```

Comment:
Pretty cool, I didn't know we had this.

Reply:
Will also be interesting to get this up on the integration test stack.

Reply:
integration-test is added